src/random.cpp

```diff
@@ -48,6 +61,51 @@ namespace libtorrent
 #endif
 			return rng;
 		}
+
+		void random_bytes(span<char> buffer)
```

Comment:
well, nor should it be. It is a safe assumption that it provides the best possible entropy source available for the system you're running on. (and, no, I don't consider old versions of gcc to matter). My understanding is that the mainstream compilers do this correctly (possibly with an exception of GCC on Windows, but if people cared about using GCC on windows, I imagine they'd fix it).